docs/workshop/workshop.rst

```diff
@@ -667,6 +670,22 @@ I praksis ser det ud som følger:
   ja
   Sag aef7ae59-e2fd-4c5d-9bc1-99bc7ad82bb9 for 'Fjernkontrol_af_SULD' lukket!
 
+
+Udtræk af eksisterende observationer til beregning
+++++++++++++++++++++++++++++++++++++++++++++++++++
+
+Som nævnt er det muligt at gå direkte fra sagsoprettelse til udtræk af observationer fra fire-databasen. Nedenfor er et eksempel på, hvordan dette kan gøres.
+
+Trin 5a) Udtræk observationer
```

Comment:
Bør det ikke stå under trin 5 hvis det er en udvidelse til det punkt?

Reply:
Jeg ovevejede dette, men lod det være et særskilt flow, ligesom jeg også beskriver det i hjælpe-teksten til kommandoen. Det er for ikke at forvirre en læser, der følger skridtene i den eksisterende og almindelige sagsgang.

Jeg retter det, så der nu står en besked i bunden af Trin 1) om, at man kan gå til enten Trin 2) eller Trin 5a) dérfra. Og så er Trin 5a) rykket op under Trin 5).